satpy/readers/fy4_base.py

```diff
@@ -169,13 +170,16 @@ def calibrate_to_bt(self, data, ds_info, ds_name):
         """Calibrate to Brightness Temperatures [K]."""
         logger.debug("Calibrating to brightness_temperature")
 
-        if self.sensor == 'AGRI':
-            lut_key = ds_info.get('lut_key', ds_name)
-        elif self.sensor == 'GHI':
-            lut_key = f'Calibration/{ds_info.get("lut_key", ds_name)}'
-        else:
-            raise ValueError(f'Unsupported sensor type: {self.sensor}')
+        if self.sensor not in ['GHI', 'AGRI']:
+            raise ValueError("Error, sensor must be GHI or AGRI.")
+
+        # The key is sometimes prefixes with `Calibration/` so we try both options here
+        lut_key = ds_info.get('lut_key', ds_name)
         lut = self.get(lut_key)
+        if lut is None:
+            lut_key = f'Calibration/{ds_info.get("lut_key", ds_name)}'
+            lut = self.get(lut_key)
```

Comment:
I think a try/except would be more pythonic:
```python
try:
   lut = self[lut_key]
except KeyError:
    lut_key = ...
    lut = self[lut_key]
```

Reply:
Done!